c/driver/postgresql/connection.cc

```diff
@@ -249,8 +282,33 @@ AdbcStatusCode PostgresConnectionGetObjectsImpl(
                  ArrowArrayAppendString(catalog_name_col, ArrowCharView(db_name)), error);
         if (depth == ADBC_OBJECT_DEPTH_CATALOGS) {
           CHECK_NA(INTERNAL, ArrowArrayAppendNull(catalog_db_schemas_col, 1), error);
-        } else {
-          return ADBC_STATUS_NOT_IMPLEMENTED;
+        } else if (!db_schema ||
```

Comment:
This is messy but there is a test that provides a schema named `this schema does not exist` and expects a NULL value back. This doesn't work if you provide a schema name that actually does exist, but neither does the sqlite implementation AFAICT

Reply:
That test expects a non-NULL, but empty, list of schemas back.  Obviously, if the schema exists, we should return it.